lib/resources/aide_conf.rb

```diff
@@ -27,7 +27,6 @@ class AideConf < Inspec.resource(1)
     include CommentParser
 
     def initialize(aide_conf_path = nil)
-      return skip_resource 'The `aide_conf` resource is not supported on your OS.' unless inspec.os.linux?
```

Comment:
We should make the supports linux here to match.

Reply:
The reason I left these as UNIX is they are fairly generic and should work across all UNIX systems.